CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## To be released
+## v2.8.2 (May 30, 2017)
 - Provided more detailed linting error message in copyright header tool
+- Remove ignore styles in base folder.
```

Comment:
This could perhaps be a bit clearer. Out of context, I'm not sure what exactly this is saying. Perhaps "Set sass linting rules to un-ignore base styles"